### ISSUE TITLE
Different color for each depth level in compactTree view

### ISSUE THREAD

**Issue by brachbach** [oughtinc/mosaic#657]
Something like this:

<img width="1223" alt="Screen Shot 2019-08-06 at 12 55 11 PM" src="https://user-images.githubusercontent.com/16952006/62572894-a14fdd80-b849-11e9-875b-932804511697.png">

**Comment by brachbach** [comment]
@stuhlmueller do you find this aesthetically offensive?

**Comment by brachbach** [comment]
@zjmiller, could you add a dev time estimate?

**Comment by stuhlmueller** [comment]
Do you think this is better than interleaving background colors, like reddit does?

<img width="500" alt="Screen Shot 2019-08-14 at 7 26 39 AM" src="https://user-images.githubusercontent.com/382515/63029169-edf87180-be64-11e9-9ee8-8fc6ba4613cd.png">

> do you find this aesthetically offensive?

Yeah, but we could probably improve the color palette and it would be okay.

**Comment by brachbach** [comment]
> Do you think this is better than interleaving background colors, like reddit does?

yeah, I think it's substantially better, because you can learn as an analyst that green = depth one, yellow = depth 2, whatever

> Yeah, but we could probably improve the color palette and it would be okay.

what sort of color palette would be better? More like earth tones?
